### ISSUE TITLE
Cannot hide "Super Favorites Menu" item in context menu

### ISSUE THREAD

**Issue by stvg88** [jmooremcc/plugin.program.super.favourites#6]
Not sure whether this is a bug or a feature request, but basically I want to hide the "Super Favorites Menu" item in context menu, since I don't know what it does/can do. Currently if I select it, it will just open the same context menu again.

I still want to keep "Add to Super Favorites" item in context menu though, that one is really helpful.

I'm using Windows 10, Kodi 19.3, with Super Favorites 2..2.5

**Comment by jmooremcc** [comment]
@stvg88 
I tried it out and it only pops up a window ID which is pointless to me.
I'll put removing it on my TO DO list for the next release.

John

**Comment by jmooremcc** [comment]
@stvg88 I've released SF vesrion 2.2.6 which resolves the Super Favorites Menu issue. The problem was caused by legacy code that no longer worked. In fact, I found that if I turned off all settings related to the context menu, the context menu was not affected and was still present. That code has now been updated and the settings menu will now control items in the context menu.

As per your original issue, turning off the "Show Super Favourites Menu Item on standard context menu" in settings will now hide that item in the context menu.
